app/views/courses/_flag_modal.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="modal fade" tabindex="-1" id="myModal" role="dialog">
```

Comment:
Do we need this file?

Reply:
No I will delete in my new branch this was from my previous test with the modals